docs/section-7/drivetrain-characterization.md

```diff
@@ -0,0 +1,83 @@
+## Introduction
+This section covers a control method known as Drivetrain Characterization, as described in [this](https://drive.google.com/drive/folders/14UI1WPtFT9KvKjAHt90VEv66ovQtFDvc) paper. In this section, we will develop the Drivetrain Characterization formula along with various tests to determine the constants needed to make the formula work.
+
+## Developing the Drivetrain Characterization Formula
+Our previous method for driving the robot used the joystick values as a percentage of the maximum voltage we want to supply to the motors. But, let's say we wanted a more accurate control method - one based on math and physics. Drivetrain Characterization allows us to more accurately determine the voltage we should supply given a desired velocity and acceleration. This section will require an understanding of the concepts described in [Part 1](https://docs.google.com/presentation/d/14PnR4rYJ5rdAwwJBectMS4YYf6jHjcI0Y-uhp9_ridM/edit#slide=id.g253f61fe15_3_0) and [Part 2](https://docs.google.com/presentation/d/1bMCfHxcyumRmsPNsWSuRBEDfwZfm4lxp9J-wn77V3Ao/edit) of our basic electronic theory presentation.
+
+Brushed DC motors work by running current through a coil of copper wires (known as the windings of the motor) as shown in the figure below. When current passes through these windings, a temporary magnetic field is created. Since there are permanent magnets in the motor, the armature (the part in the center connected to the coil of wires that is allowed to rotate) begins to accelerate towards the magnets (the north end of the armature is attracted to the positive end and the south end is attracted to the negative end). As well, when the armature's acceleration becomes zero, the flow of current reverses and the north end of the armature rotates towards the positive end and the south end rotates towards the negative end. This process repeats as long as there is voltage across the motor.
```

Comment:
"positive end" -> "south pole of the permanent magnet"
"negative end" -> "north pole of the permanent magnet"

In the picture/animation you are using, the north poles are purple and the south poles are pink and the picture emphasis that the like poles repel, so it might make more sense to describe it that way.

Also, "when the armature's acceleration becomes zero, the flow of current..." -> "when the armature rotates so that the opposite poles are near each other, the yellow and blue wires are connected to the opposite ends of the coil. As a result, the flow of current..."